torchvision/transforms/v2/_transform.py

```diff
@@ -28,6 +30,10 @@ def _check_inputs(self, flat_inputs: List[Any]) -> None:
     def _get_params(self, flat_inputs: List[Any]) -> Dict[str, Any]:
         return dict()
 
+    def _call_or_noop(self, dispatcher: Callable, inpt: Any, *args: Any, **kwargs: Any) -> Any:
```

Comment:
I don't think we need this. We can just inline `_get_kernel(dispatcher, type(inpt), allow_passthrough=True)(...)` everwhere.

Reply:
Fair enough. I still have a slight pref for inlining because it avoids an extra call. If we go back to the helper I'd suggest just calling it `call_kernel`. The fact that some kernels are noop is irrelevant for its name IMO.